__tests__/app.test.js

```diff
@@ -101,3 +102,69 @@ describe("GET /api/articles/:articleid", () => {
       });
   });
 });
+
+describe.only("GET /api/articles", () => {
+  test("responds with status code 200", () => {
+    return request(app).get("/api/articles").expect(200);
+  });
+  test("responds with an articles array", () => {
+    return request(app)
+      .get("/api/articles")
+      .then(({ body }) => {
+        expect(Array.isArray(body.articles)).toBe(true);
+      });
+  });
+  test("responds with an articles array of objects containing the correct properties", () => {
+    return request(app)
+      .get("/api/articles")
+      .then(({ body }) => {
+        const articlesArray = body.articles;
+        expect(
+          articlesArray.every((article) => article.hasOwnProperty("author"))
+        ).toBe(true);
+        expect(
+          articlesArray.every((article) => article.hasOwnProperty("title"))
+        ).toBe(true);
+        expect(
+          articlesArray.every((article) => article.hasOwnProperty("article_id"))
+        ).toBe(true);
+        expect(
+          articlesArray.every((article) => article.hasOwnProperty("topic"))
+        ).toBe(true);
+        expect(
+          articlesArray.every((article) => article.hasOwnProperty("created_at"))
+        ).toBe(true);
+        expect(
+          articlesArray.every((article) => article.hasOwnProperty("votes"))
+        ).toBe(true);
+        expect(
+          articlesArray.every((article) =>
+            article.hasOwnProperty("article_img_url")
+          )
+        ).toBe(true);
+        expect(
+          articlesArray.every((article) =>
+            article.hasOwnProperty("comment_count")
+          )
+        ).toBe(true);
+        expect(
+          articlesArray.every((article) => article.hasOwnProperty("body"))
+        ).toBe(false);
+      });
```

Comment:
You may want to look into objectContaining ( e.g...
expect.objectContaining({
article_id: expect.any(Number),
...

This will allow you to check the types of the values as well as saving you some space :)